RevenueCatUI/Views/TimelineView.swift

```diff
@@ -0,0 +1,322 @@
+//
+//  File.swift
+//  
+//
+//  Created by Guillaume LAURES on 15/05/2024.
+//
+
+import SwiftUI
+import UIKit
+
+extension NSLayoutConstraint.Axis {
+    
+    var opposite: NSLayoutConstraint.Axis {
+        switch self {
+        case .vertical:
+            return .horizontal
+        case .horizontal:
+            return .vertical
+        @unknown default:
+            return self
+        }
+    }
+}
+
+@available(iOS 13.0, *)
+extension HorizontalAlignment {
+    private enum IconAlignment: AlignmentID {
+        static func defaultValue(in dimensions: ViewDimensions) -> CGFloat {
+            return dimensions[HorizontalAlignment.center]
+        }
+    }
+    static let iconAlignment = HorizontalAlignment(IconAlignment.self)
+}
```

Comment:
that's somewhat opinionated - can you move this code to the bottom of the file or in a separate altogether? It looks more readable if the entry-point is top-most.

Reply:
Also made them private.